src/test/java/com/toopher/TestToopherAPI.java

```diff
@@ -1,89 +1,363 @@
-                "{'id':'1','enabled':true,'pending':true,'user':{'id':'1','name':'some user'}}".replace("'", "\""));
+    public void testCreatePairingWithPairingPhrase() throws InterruptedException, RequestError {
+        HttpClientMock httpClient = new HttpClientMock(200, pairingJsonResponse.toString());
+        ToopherApi toopherApi = new ToopherApi("key", "secret",
+                createURI(DEFAULT_BASE_URL), httpClient);
+        Pairing pairing = toopherApi.pair(userName, "awkward turtle");
+
+        assertEquals("POST", httpClient.getLastCalledMethod());
+        assertEquals("pairings/create", httpClient.getLastCalledEndpoint());
+        assertEquals("awkward turtle", httpClient.getLastCalledData("pairing_phrase"));
+        assertEquals(userName, httpClient.getLastCalledData("user_name"));
+        assertEquals(id, pairing.id);
+        assertEquals(userId, pairing.user.id);
+    }
 
-        ToopherAPI toopherApi = new ToopherAPI("key", "secret",
-                createURI("https://api.toopher.test/v1"), httpClient);
-        PairingStatus pairing = toopherApi.pair("awkward turtle", "some user");
+    @Test
+    public void testCreateQrPairing() throws InterruptedException, RequestError {
+        HttpClientMock httpClient = new HttpClientMock(200, pairingJsonResponse.toString());
+        ToopherApi toopherApi = new ToopherApi("key", "secret",
+                createURI(DEFAULT_BASE_URL), httpClient);
+        Pairing pairing = toopherApi.pair(userName);
 
-        assertEquals(httpClient.getLastCalledMethod(), "POST");
-        assertEquals(httpClient.getLastCalledData("pairing_phrase"), "awkward turtle");
-        
-        assertEquals(pairing.userId, "1");
-        assertEquals(pairing.userName, "some user");
-        assertTrue(pairing.pending);
-        assertTrue(pairing.enabled);
+        assertEquals("POST", httpClient.getLastCalledMethod());
+        assertEquals("pairings/create/qr", httpClient.getLastCalledEndpoint());
+        assertEquals(userName, httpClient.getLastCalledData("user_name"));
+        assertEquals(id, pairing.id);
+        assertEquals(userId, pairing.user.id);
     }
 
     @Test
-    public void testGetPairingStatus() throws InterruptedException, RequestError {
-        HttpClientMock httpClient = new HttpClientMock(200,
-                "{'id':'1','enabled':true,'pending':true,'user':{'id':'1','name':'some user'}}".replace("'", "\""));
+    public void testCreateSmsPairing() throws InterruptedException, RequestError {
+        HttpClientMock httpClient = new HttpClientMock(200, pairingJsonResponse.toString());
 
-        ToopherAPI toopherApi = new ToopherAPI("key", "secret",
-                createURI("https://api.toopher.test/v1"), httpClient);
-        PairingStatus pairing = toopherApi.getPairingStatus("1");
+        ToopherApi toopherApi = new ToopherApi("key", "secret",
+                createURI(DEFAULT_BASE_URL), httpClient);
+        Pairing pairing = toopherApi.pair(userName, "123456");
```

Comment:
Can you use a string that looks more like a real phone number? E.g., `"5245678"` or `"5551234"`